tests/output/sapi_windows_vt100_support_stream_lose_data.phpt

```diff
@@ -0,0 +1,40 @@
+--TEST--
+sapi_windows_vt100_support(): casting stream emits data loss
+--SKIPIF--
+<?php
+if(PHP_OS_FAMILY !== "Windows") {
+    echo "skip Only for Windows systems";
+} elseif (version_compare(
+    PHP_WINDOWS_VERSION_MAJOR.'.'.PHP_WINDOWS_VERSION_MINOR.'.'.PHP_WINDOWS_VERSION_BUILD,
+     '10.0.10586'
+) >= 0) {
+    echo "skip Only for Windows systems < 10.0.10586";
```

Comment:
Why is the test skipped on more recent Windows versions?

Reply:
I copied the skipif section from another test, they all looked to skip them as of that version not sure why.

Reply:
Ah, I see! These tests come in pairs (\*\_winok\_\*.phpt and \*\_winko\_\*.phpt); the latter are for older Windows versions, and the former for newer Windows, but that difference is only about `ENABLE_VIRTUAL_TERMINAL_PROCESSING` (which enables parsing of control character sequences), and this is only supported as of 10586. I don't think this is relevant for the new tests, so that part of the SKIPIF can be removed.